examples/nuxt-openai/app.vue

```diff
@@ -1,7 +1,9 @@
-<script setup>
+<script setup lang="ts">
 import { useChat } from 'ai/vue'
 
-const { messages, input, handleInputChange, handleSubmit } = useChat()
+const { messages, input, handleSubmit } = useChat({
+  headers: { 'Content-Type': 'application/json' }
```

Comment:
It is required? Also not sure if there is any config but my local test was making constant requests to `/chat-0-loading` (basically loading `App.vue`!) dummy solution: https://github.com/pi0/nuxt-openai-demo/blob/main/server/routes/chat-0-loading.ts

Reply:
I don't know why that's happening. Yes the headers is required otherwise `readBody` returns a string instead of parsed JSON.

Reply:
@danielroe By any chance do you have more insight about `chat-0-loading` ?

Reply:
yes, I'll raise a separate PR for this